README.md

```diff
@@ -97,7 +97,8 @@ Inspired by the [awesome](#more-awesome) list thing. Feel free to <a href="https
 
 * [Elmish](https://elmish.github.io/elmish/) - Elm-like abstractions for F# apps
 * [Elmish.Bridge](https://github.com/Nhowka/Elmish.Bridge) - Create client-server Fable-Elmish apps keeping a single mindset
-* [Fable.PowerPack](https://github.com/fable-compiler/fable-powerpack) - Utilities for Fable apps
+* [Fable.Fetch](https://github.com/fable-compiler/fable-fetch) - Fable bindings for Browsers' Fetch API.
+* [Fable.Promise](https://github.com/fable-compiler/fable-promise) - Fable bindings for JS promise.
 * [Fable.Remoting](https://github.com/Zaid-Ajaj/Fable.Remoting) - Typed RPC client-server communication for Fable and .NET
```

Comment:
Can you please add `Fable.Date` too?

https://github.com/fable-compiler/fable-date